platform-coop-toolkit/coop-map-directory-index
# feat: replace Font Awesome with Pinecone icons

## Description

Replaces Font Awesome with Pinecone icons.

## Steps to test

1. Check out branch.
2. `npm install`
3. `npm run build:production`
4. Update Social network icon fields to match Pinecone icon names (see: https://github.com/platform-coop-toolkit/pinecone/blob/dev/src/components/10-atoms/20-icons/icons.config.js#L208-L234)

**Expected behavior:** Social account icons appear where expected.

## Additional information

Not applicable.

## Related issues

Not applicable.
